courses/free-courses-hi.md

```diff
@@ -318,10 +318,12 @@
 
 ### JavaScript
 
+* [Chai aur Javascript | हिन्दी](https://www.youtube.com/playlist?list=PLu71SKxNbfoBuX3f4EOACle2y-tRC5Q37) - Chai aur Code
```

Comment:
```suggestion
* [Chai aur Javascript \| हिन्दी](https://www.youtube.com/playlist?list=PLu71SKxNbfoBuX3f4EOACle2y-tRC5Q37) - Chai aur Code
```

Reply:
when you've made changes, please add `(changes made)` to the title of your PR

Reply:
Ok, will keep this in mind form now onwards.